jenkins/bootstrap_test.py

```diff
@@ -836,6 +835,24 @@ def testBranch(self):
         os.chdir('/tmp')
         bootstrap.bootstrap('fake-branch', self.REPO, self.BRANCH, None, self.root_workspace)
 
+    def testBatch(self):
+        def get_ref():
+            return subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
+        refs = ['master:%s' % get_ref()]
+        for pr in (123, 456):
+            subprocess.check_call(['git', 'checkout', '-b', 'refs/pull/%d/head' % pr, 'master'])
+            subprocess.check_call(['git', 'rm', self.MASTER])
+            subprocess.check_call(['touch', self.PR_FILE])
+            subprocess.check_call(['git', 'add', self.PR_FILE])
+            open('pr_%d.txt' % pr, 'w').write('some text')
+            subprocess.check_call(['git', 'add', 'pr_%d.txt' % pr])
+            subprocess.check_call(['git', 'commit', '-m', 'add some stuff (#%d)' % pr])
+            refs.append('%d:%s' % (pr, get_ref()))
+        os.chdir('/tmp')
+        pull = ','.join(refs)
```

Comment:
I would note that we are doing it this way because we don't know the shas ahead of time (since this would be easier to read as `master:abcd,123:aaa,444:bbb`